specs/clickhouse/clickhouse.spec

```diff
@@ -57,24 +57,21 @@
 
 Summary:           Yandex ClickHouse DBMS
 Name:              clickhouse
-Version:           19.9.4.34
+Version:           19.14.7.15
 Release:           0%{?dist}
 License:           APL 2.0
 Group:             Applications/Databases
 URL:               https://clickhouse.yandex
 
 Source:            %{name}-%{version}.tar.bz2
-Source1:           %{name}.logrotate
-
-Patch0:            %{name}-fix-tz-filepath.patch
```

Comment:
Patch removed from spec but still exists in the repository.

Reply:
This patch is already merged to the code base. I forgot to delete it from the `SOURCES` directory. Fixed (d9a58dc).